### PR TITLE
Andream16/refresh docs

### PR DESCRIPTION
**Pull request by andream16**
- Linked Discord Server in README
- Refreshed our getting started docs to get started with Dracon and get value as soon as possible and be more friendly to non k8s experts
- Moved more complex parts about non-standard setup into a separated document
- Moved building new components into a separated document
- Fixed a bug in the example Pipeline provided on the build pipeline doc

### CODE REVIEW

File: docs/custom-components.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
+# Building Custom Components
+
+## Deploying a custom version of Dracon components
+
+So, the first step is to build all the containers and push them to a registry
```
Comment (northdpole) [inline docs/custom-components.md:5]: nit: consider removing the "So," , it's a matter of speaking not writing

File: docs/custom-components.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
+# Building Custom Components
```
Comment (northdpole) [inline docs/custom-components.md:1]: is this missing the `scripts/kind-with-registry` part?

**Review by northdpole — APPROVED** [review]

**Review by northdpole — APPROVED** [review]

File: docs/writing-pipelines.md [hunk 3]
```diff
@@ -35,19 +35,18 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 nameSuffix: -go-pipeline
 resources:
-  - ../../../components/base/pipeline.yaml
-  - ../../../components/base/task.yaml
+  - /components/base/pipeline.yaml
```
Comment (ptzianos) [inline docs/writing-pipelines.md:38]: I don't think these resources are needed any more, they should be injected automatically by the execution engine

File: docs/writing-pipelines.md [hunk 4]
```diff
@@ -35,19 +35,18 @@ apiVersion: kustomize.config.k8s.io/v1beta1
...
-  - ../../../components/producers/aggregator
-  - ../../../components/producers/golang-gosec
-  - ../../../components/producers/golang-nancy
-  - ../../../components/enrichers/aggregator
-  - ../../../components/enrichers/policy
-  - ../../../components/enrichers/deduplication
-  - ../../../components/consumers/mongodb
-  - ../../../components/consumers/elasticsearch
-
+  - /components/sources/git
```
Comment (ptzianos) [inline docs/writing-pipelines.md:41]: wont' the `/` in the beginning cause an issue when the path is merged?

Comment (andream16) [reply docs/writing-pipelines.md:41]: When tested locally while building the atom enricher this did the trick :)

File: docs/writing-pipelines.md (hunk 3 above)
Comment (andream16) [reply docs/writing-pipelines.md:38]: Good spot

**Review by ptzianos — APPROVED** [review]

**Pull request merged**